### PR TITLE
Added missing undraws + download automatically

### PR DESCRIPTION
**Pull request by andreimc**
## Description
Hey man I started writing this thing before I saw yours and I added some extra features and missing items

## Checklist
Please ensure your pull request fulfills the following requirements:

- [x] The commit messages follow our guidelines ([CONTRIBUTING.md](./CONTRIBUTING.md)).
- [x] Tests for any changes have been added (for bug fixes / features).
- [x] Docs have been added / updated (for bug fixes / features).

## Type
What kind of change does this pull request introduce?

```
[ ] Bug.
[x] Feature.
[ ] Code style update (formatting, local variables).
[ ] Refactoring (no functional changes, no api changes).
[ ] Build related changes.
[ ] CI related changes.
[ ] Documentation content changes.
[ ] Other (please describe below).
```

## Breaking Changes
Does this pull request introduce any breaking changes?

```
[ ] Yes
[x] No
```

## Other Information

Ran them trough prettierrc removed the ignore, cause it gets written with tabs for some reason.

### CODE REVIEW

File: scripts/generate-mappings.js [hunk 1]
```diff
@@ -7,6 +7,26 @@ const path = require('path');
 
 const illustrations = require('./illustrations.json');
 
+function generateExports() {
```
Comment (justinlettau) [inline scripts/generate-mappings.js]: Could we move the generation of the barrel (index.js) to it's own scripts, to keep separation of concerns?

File: scripts/download.js [hunk 2]
```diff
@@ -2,17 +2,48 @@ const axios = require('axios');
...
 const ora = require('ora');
 const path = require('path');
+const rp = require('request-promise');
+const $ = require('cheerio');
 
 const illustrations = require('./illustrations.json');
 
+const url = 'https://42f2671d685f51e10fc6-b9fcecea3e50b3b59bdc28dead054ebc.ssl.cf5.rackcdn.com/illustrations';
+
+function buildIlustrationsJson() {
```
Comment (justinlettau) [inline scripts/download.js]: Could we move the generation of the json file to it's own scripts, to keep separation of concerns?

File: scripts/download.js [hunk 3]
```diff
@@ -2,17 +2,48 @@ const axios = require('axios');
...
+  const crawlUrl = index => `https://undraw.co/illustrations/load/${index}`;
+  const promises = [];
+  const data = [];
+
+  for (let i = 0; i < 35; i++) {
+    promises.push(
+      rp(crawlUrl(i))
+        .then(function(html) {
```
Comment (justinlettau) [inline scripts/download.js]: Trying to stay consistent, can we change these `function`s to `=>` type functions?

File: scripts/download.js [hunk 4]
```diff
@@ -2,17 +2,48 @@ const axios = require('axios');
...
 const illustrations = require('./illustrations.json');
 
+const url = 'https://42f2671d685f51e10fc6-b9fcecea3e50b3b59bdc28dead054ebc.ssl.cf5.rackcdn.com/illustrations';
+
+function buildIlustrationsJson() {
+  const crawlUrl = index => `https://undraw.co/illustrations/load/${index}`;
+  const promises = [];
+  const data = [];
+
+  for (let i = 0; i < 35; i++) {
```
Comment (justinlettau) [inline scripts/download.js]: Curious: where does the `35` come from? Will this need updated in the future as the number of illustrations grow?

File: package.json [hunk 5]
```diff
@@ -44,7 +44,8 @@
     "lib": "npm run lib:clean && npm run lib:build",
     "lib:clean": "rimraf dist",
     "lib:build": "cross-env NODE_ENV=production babel src/lib --out-dir dist --copy-files",
-    "format:all": "prettier --write **/*.js"
+    "format:all": "prettier --write **/*.js",
+    "format:illustrations": "prettier --write src/lib/illustrations/*.js"
```
Comment (justinlettau) [inline package.json]: Curious: why the addition of the extra script here? Doesn't the `format:all` cover these files too?

**Comment by justinlettau** [comment]
@andreimc I like the automation! 🎉 Thanks for this!

File: package.json (hunk 5 above)
Comment (andreimc) [reply package.json]: nop it doesn't seem to recurrse

File: scripts/download.js (hunk 4 above)
Comment (andreimc) [reply scripts/download.js]: it's just arbitrary they are on page 31 now keeps adding can increase later

**Comment by andreimc** [comment]
@justinlettau Addressed your other concerns

**Pull request merged**